test-manager.go

```diff
@@ -19,6 +19,21 @@ import (
 	"github.com/rocket-pool/node-manager-core/eth"
 )
 
+// Interface representing individual module snapshots that compose an entire Snapshot
+type IOshaModule interface {
```

Comment:
This and the `Snapshot` struct should probably be in their own file(s) instead of here in the test manager.